mountaineer/app.py

```diff
@@ -79,6 +79,11 @@ def __init__(
             *(custom_builders if custom_builders else []),
         ]
 
+        # If this flag is present, we will re-raise this error during render()
+        # so users can see the error in the browser.
+        # This is useful for debugging, but should not be used in production
+        self.build_exception: Exception | None = None
```

Comment:
This flag is only set by our local `runserver` isolated process, which is only used during local development. In production uvicorn scenarios this will maintain its default value so no error will be thrown at render-time.